### PR TITLE
[WIP] Monitoring RBAC matrix audit

### PR DESCRIPTION
**Pull request by DonNateR**
- Fix issue #1707 

SME review already implemented with PR #1810.

### CODE REVIEW

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-21185feec9/how-to/).

**Comment by kmsholcomb** [comment]
Hi @DonNateR. See attached edits. Lots of nitpicky markup on this one!

[Detailed permissions matrix for Rackspace Monitoring_editsKH.pdf](https://github.com/rackerlabs/rackspace-how-to/files/748909/Detailed.permissions.matrix.for.Rackspace.Monitoring_editsKH.pdf)
[Detailed permissions matrix for Rackspace Monitoring_editsKH.docx](https://github.com/rackerlabs/rackspace-how-to/files/748908/Detailed.permissions.matrix.for.Rackspace.Monitoring_editsKH.docx)

**Comment by DonNateR** [comment]
Redirect needed for this PR before merging.

File: content/rackspace-monitoring/detailed-permissions-matrix-for-rackspace-monitoring.md [hunk 1]
```diff
@@ -1,111 +1,215 @@
...
+
+### Check operations
+
+Method 	| API action | Role | Description
+--- | --- | --- | ---
+List checks for an entity | `GET /v1.0/entities/{entityId}/checks` | **Creator, Admin** | Lists the checks associated with a given entityId.
+Get a check by ID | `GET /v1.0/entities/{entityId}/checks/{checkId}` | **Creator, Admin** | Returns information about the specified check.
+Update a check by ID | `PUT /v1.0/entities/{entityId}/checks/{checkId}` | **Admin** | Updates the specified check.
+Delete a check by ID | `DELETE /v1.0/entities/{entityId}/checks/{checkId}` | **Admin** | Deletes the specfied check from your account.
+Create a check | `POST /v1.0/entities/{entityId}/checks` | **Creator, Admin** | Creates a new check and associates it with an entity using the parameters listed in [Attributes](https://developer.rackspace.com/docs/cloud-monitoring/v1/developer-guide/#attributes).
```
Comment (nimbinatus) [inline content/rackspace-monitoring/detailed-permissions-matrix-for-rackspace-monitoring.md]: Recommend replacing the link here with https://developer.rackspace.com/docs/rackspace-monitoring/v1/api-reference/check-operations/#check-operations (more specific to the list noted). Also, I'd change `[Attributes]` to `[Attributes used for all checks]`.

File: content/rackspace-monitoring/detailed-permissions-matrix-for-rackspace-monitoring.md [hunk 2]
```diff
@@ -1,111 +1,215 @@
...
+Test an existing check | `POST /v1.0/entities/{entityId}/checks/{checkId}/test` | **Observer, Creator, Admin** | Tests a check inline.
+Test a check | `POST /v1.0/entities/{entityId}/test-check` | **Observer Creator, Admin** | Tests a check before you create it.
+
+
+### Check type operations
+
+Method 	| API action | Role | Description
+--- | --- | --- | ---
+List check types | `GET /v1.0/check_types` | **Observer, Creator, Admin** | List all the available check types.
+Get a check type by ID| `GET /v1.0/check_types/{checkTypeId}` | **Observer, Creator, Admin** | Retrieves information for the specified check type.
```
Comment (nimbinatus) [inline content/rackspace-monitoring/detailed-permissions-matrix-for-rackspace-monitoring.md]: space needed before the first `|`

File: content/rackspace-monitoring/detailed-permissions-matrix-for-rackspace-monitoring.md [hunk 3]
```diff
@@ -1,111 +1,215 @@
...
+
+
+### Notification plans operations
+
+Method 	| API action | Role | Description
+--- | --- | --- | ---
+List notification plans | `GET /v1.0/notification_plans` | **Observer, Creator, Admin** | Lists the notification plans for the account.
+Get a notification plan by ID | `GET /v1.0/notification_plans/{notificationPanId}` | **Observer, Creator, Admin** | Gets information about the specified notification plan.
+Update a notification plan by ID | `PUT /v1.0/notification_plans/{notificationPanId}` | **Admin** | Updates the specified notification plan.
+Delete a notification plan| `DELETE /v1.0/notification_plans/{notificationPanId}` | **Admin** | Deletes the specified notification plan.
```
Comment (nimbinatus) [inline content/rackspace-monitoring/detailed-permissions-matrix-for-rackspace-monitoring.md]: space needed before the first `|`

File: content/rackspace-monitoring/detailed-permissions-matrix-for-rackspace-monitoring.md [hunk 4]
```diff
@@ -1,111 +1,215 @@
...
+Create a notification plan | `POST /v1.0/notification_plans` | **Creator, Admin** | Creates a notification plan.
+
+
+
+### Suppressions operations
+
+Method 	| API action | Role | Description
+--- | --- | --- | ---
+List suppressions | `GET /v1.0/suppressions` | **Observer, Creator, Admin** | Returns a list of suppressions.
+Get a suppression by ID| `GET /v1.0/suppressions/{suppressionId}` | **Observer, Creator, Admin** | Gets details of the specified suppression.
```
Comment (nimbinatus) [inline content/rackspace-monitoring/detailed-permissions-matrix-for-rackspace-monitoring.md]: space needed before the first `|`

File: content/rackspace-monitoring/detailed-permissions-matrix-for-rackspace-monitoring.md [hunk 5]
```diff
@@ -1,111 +1,215 @@
...
+Update a suppression | `PUT /v1.0/suppressions/{suppressionId}` | **Admin** | Updates the specified suppression.
+Delete a suppression| `DELETE /v1.0/suppressions/{suppressionId}` | **Admin** | Deletes the specified suppression.
```
Comment (nimbinatus) [inline content/rackspace-monitoring/detailed-permissions-matrix-for-rackspace-monitoring.md]: space needed before the first `|`

File: content/rackspace-monitoring/detailed-permissions-matrix-for-rackspace-monitoring.md [hunk 6]
```diff
@@ -1,111 +1,215 @@
...
+Create an alarm | `POST /v1.0/entities/{entityId}/alarms` | **Creator, Admin** | Creates a new alarm for the specified entity.
+Test an alarm | `POST /v1.0/entities/{entityId}/test-alarm` | **Observer, Creator, Admin** | Tests whether the alarm criteria is valid and shows how the alarm state is evaluated.
+
+
+### Alarm examples operations
+
+Method 	| API action | Role | Description
+--- | --- | --- | ---
+List alarm examples | `GET /v1.0/alarm_examples` | **Observer, Creator, Admin** | Returns a list of alarm examples.
+Get alarm example by ID| `GET /v1.0/alarm_examples/{alarmExampleId}` | **Observer, Creator, Admin** | Returns information about the specified alarm example.
```
Comment (nimbinatus) [inline content/rackspace-monitoring/detailed-permissions-matrix-for-rackspace-monitoring.md]: space needed before the first `|`

File: content/rackspace-monitoring/detailed-permissions-matrix-for-rackspace-monitoring.md [hunk 7]
```diff
@@ -1,111 +1,215 @@
...
+Create agent token | `POST /v1.0/agent_tokens` | **Observer, Creator, Admin** | Creates a new agent token.
+
+
+### Alarm operations
+
+Method 	| API action | Role | Description
+--- | --- | --- | ---
+List alarms | `GET /v1.0/entities/{entityId}/alarms` | **Observer, Creator, Admin** | Lists the alarms on the specified entity.
+Get alarm by ID | `GET /v1.0/entities/{entityId}/alarms/{alarmId}` | **Observer, Creator, Admin** | Gets information about the specified single alarm.
+Update alarm by ID| `PUT /v1.0/entities/{entityId}/alarms/{alarmId}` | **Admin** | Updates an alarm with the specified alarmId. Partial updates to an alarm are acceptable. You may specify only the parameters you would like to update.
```
Comment (nimbinatus) [inline content/rackspace-monitoring/detailed-permissions-matrix-for-rackspace-monitoring.md]: space needed before the first `|`

File: content/rackspace-monitoring/detailed-permissions-matrix-for-rackspace-monitoring.md [hunk 8]
```diff
@@ -1,111 +1,215 @@
...
+The matrix displays the Rackspace Monitoring methods groups by category, their corresponding RESTful API commands, and the RBAC roles that are supported.
+
+
+### Account operations
+
+Method | API action | Role | Description
+--- | --- | --- | ---
+Get account information | `GET /v1.0/account` | **Observer, Creator, Admin** | Returns account information.
+Update properties on an account| `PUT /v1.0/account` | **Admin** | Updates properties on an account.
+Get audit information	| `GET /v1.0/audits` | **Observer, Creator, Admin** | Returns audits information for the account.
```
Comment (nimbinatus) [inline content/rackspace-monitoring/detailed-permissions-matrix-for-rackspace-monitoring.md]: Are there extra spaces before the first `|`?

**Comment by nimbinatus** [comment]
Ready to merge

**Pull request merged**